composer.json

```diff
@@ -14,8 +14,8 @@
     "laminas/laminas-cache-storage-implementation": "1.0"
   },
   "require-dev": {
-    "laminas/laminas-cache": "3.0.x-dev",
-    "laminas/laminas-cache-storage-adapter-test": "2.0.x-dev",
+    "laminas/laminas-cache": "3.1.x",
```

Comment:
@boesing given our quicker release iterations, do we need to use `-dev` packages at all here?

Reply:
Nope, these are still there from the time before I created a 3.0 release of `laminas-cache`. Will remove these now one-by-one